src/assets/scss/app.scss

```diff
@@ -0,0 +1,19 @@
+@import "~bulma";
+@import "~bulma-timeline";
+
```

Comment:
Bulma is a free, open source CSS framework based on flexbox. It simplifies layout creation process and has a fairly simple syntax. In this case, in order to reduce the size of our app, we could either import only the Bulma tools that we need or simply use its CDN link.

Reply:
In order to  use SASS in our application, we first need to add node-sass script by typing in  `yarn add node-sass ` command, otherwise it won’t compile properly resulting in errors and messy layout.